cumulus/test/service/benches/validate_block_glutton.rs

```diff
@@ -96,7 +96,9 @@ fn benchmark_block_validation(c: &mut Criterion) {
 		runtime.block_on(import_block(&client, parachain_block.blocks()[0].clone(), false));
 		let runtime = utils::get_wasm_module();
 
-		let sproof_builder: RelayStateSproofBuilder = Default::default();
+		let mut sproof_builder: RelayStateSproofBuilder = Default::default();
+		sproof_builder.num_authorities = 2;
```

Comment:
Weird that you need to set this here and above you construct the block without the descendants.